tests/Benchmarks/System.Text.JsonLab/Comparison/DataGenerator.cs

```diff
@@ -55,31 +72,56 @@ private static IndexViewModel CreateIndexViewModel()
                     ImageUrl = "https://www.dotnetfoundation.org/theme/img/carousel/foundation-diagram-content.png",
                     Title = "Promoting Open Source"
                 },
-                ActiveOrUpcomingEvents = Enumerable.Repeat(
```

Comment:
FYI, @adamsitnik, we should probably make a similar change in the performance repo: https://github.com/dotnet/performance/blob/master/src/benchmarks/Serializers/DataGenerator.cs